src/Platform.Handlers/src/Xamarin.Platform.Handlers/PropertyMapper.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
the mount of Linq, and collection handling, will need to be evaluated for performance on devices